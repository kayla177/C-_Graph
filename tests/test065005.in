ENTITY TDGXZWPM name_TDGX paper
ENTITY SYTFMEH7 name_SYTF institution
ENTITY HS08MT0W name_HS08 journal
ENTITY QKPD8D07 name_QKPD journal
ENTITY 8CP2LXQM name_8CP2 journal
ENTITY SDKK387P name_SDKK institution
ENTITY WETK9KRU name_WETK author
ENTITY U4MH233M name_U4MH journal
ENTITY DU4VYO2B name_DU4V journal
ENTITY XYY15PFS name_XYY1 institution
RELATIONSHIP QKPD8D07 affiliated_with HS08MT0W 6.3
RELATIONSHIP WETK9KRU affiliated_with SDKK387P 0.7
RELATIONSHIP XYY15PFS affiliated_with DU4VYO2B 8.7
RELATIONSHIP TDGXZWPM published_in U4MH233M 3.5
RELATIONSHIP DU4VYO2B published_in HS08MT0W 8.3
RELATIONSHIP U4MH233M authored_by QKPD8D07 8.3
RELATIONSHIP SYTFMEH7 published_in WETK9KRU 9.5
RELATIONSHIP SDKK387P authored_by TDGXZWPM 0.2
RELATIONSHIP DU4VYO2B published_in SDKK387P 2.6
RELATIONSHIP 8CP2LXQM affiliated_with SDKK387P 3.8
RELATIONSHIP WETK9KRU cited_by 8CP2LXQM 9.3
RELATIONSHIP WETK9KRU cited_by TDGXZWPM 6.9
RELATIONSHIP DU4VYO2B affiliated_with QKPD8D07 3.5
RELATIONSHIP WETK9KRU presented_at XYY15PFS 3.5
RELATIONSHIP SYTFMEH7 published_in XYY15PFS 3.6
PRINT WETK9KRU
HIGHEST
PATH WETK9KRU SDKK387P
PRINT HS08MT0W
DELETE QKPD8D07
PATH WETK9KRU U4MH233M
HIGHEST
PATH WETK9KRU 8CP2LXQM
DELETE DU4VYO2B
PATH SYTFMEH7 U4MH233M
EXIT
