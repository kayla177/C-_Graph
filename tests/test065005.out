success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
SDKK387P SYTFMEH7 8CP2LXQM TDGXZWPM XYY15PFS
U4MH233M XYY15PFS 31.6
WETK9KRU 8CP2LXQM SDKK387P 13.1
QKPD8D07 DU4VYO2B
success
WETK9KRU TDGXZWPM U4MH233M 10.4
HS08MT0W U4MH233M 30.9
WETK9KRU 8CP2LXQM 9.3
success
SYTFMEH7 WETK9KRU TDGXZWPM U4MH233M 19.9
